### DISCUSSION TITLE
Welcome to test-achievements Discussions!

### DISCUSSION THREAD

**Discussion by LuckyGardel** [LuckyGardel/test-achievements#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LuckyGardel** [comment]
this is the right answer

**Reply by fabio1623** [reply]
I agree

**Reply by Xamweis** [reply]
no

**Reply by LuckyGardel2** [reply]
daaaaaaaamm

**Comment by Xamweis** [comment]
hi

**Reply by LuckyGardel** [reply]
whaaaat

**Comment by fabio1623** [comment]
Whatever is the new comment

**Reply by LuckyGardel** [reply]
sure

**Reply by LuckyGardel2** [reply]
hmmm
